bin/test.py

```diff
@@ -0,0 +1,282 @@
+#!/usr/bin/env python
+
+import boto3, botocore, os, time, sys
+from os.path import expanduser
+from botocore.client import Config
+
+TEST_APP_BUCKET_PREFIX='awslabs-startup-kit-templates-test-eb-v1-tmp-'
+TEST_APP_SOURCE_BUCKET='awslabs-startup-kit-templates-test-eb-v1'
+TEST_PYTHON_APP_KEY='eb-python-flask.zip'
+STACK_BUCKET='awslabs-startup-kit-templates-deploy-v1'
+EB_TEMPLATE_URL='https://s3.amazonaws.com/awslabs-startup-kit-templates-deploy-v1/vpc-bastion-eb-rds.cfn.yml'
+VPC_TEMPLATE_URL='https://s3.amazonaws.com/awslabs-startup-kit-templates-deploy-v1/vpc.cfn.yml'
+VPC_BASTION_TEMPLATE_URL='https://s3.amazonaws.com/awslabs-startup-kit-templates-deploy-v1/vpc-bastion.cfn.yml'
+KEY_PAIR_PREFIX='sktemplates-test-'
+
+# TODO Add support for Fargate test in supported regions
+# TODO Delete the RDS snapshots and any other resources
+# TODO ssh into bastion host to validate
+# TODO Make http requests to endpoints to validate
+# TODO Connect to database through bastion to validate
+# TODO Store key pair temporarily and then cleanup
```

Comment:
An awful lot of TODOs on a multiple-hundred-line script further points my nose in the "decompose" direction.

Reply:
Totally. I will get these into SIM and out of the repo in the next release.